README.md

```diff
@@ -17,8 +17,8 @@ editor.
 
 # AUTHORS
 
-*   Abdeali J Kothari - <mailto:abdealikothari@gmail.com>
-*   Lasse Schuirmann - <mailto:lasse.schuirmann@gmail.com>
+- Abdeali J Kothari - <mailto:abdealikothari@gmail.com>
```

Comment:
sorry for not responding for so long, let's just ignore gitmate on this one then